gradle-plugin/src/functionalTest/kotlin/io/spine/protodata/gradle/plugin/PluginSpec.kt

```diff
@@ -197,6 +199,12 @@ class PluginSpec {
             //.enableRunnerDebug()
             .copyBuildSrc()
         project = builder.create()
+        (project.runner as DefaultGradleRunner).withJvmArguments(
```

Comment:
I suggest to have these settings in `gradle.properties` file _inside_ the test-resource directory instead. In this way, it would be easier to locate all similar settings, e.g. via searching by `org.gradle.jvmargs=` string in IDE/GitHub.

Reply:
This is now done to re-use the same configuration for 8 testing sub-projects. So leaving as-is.